query-connector/src/app/CustomQuery.ts

```diff
@@ -75,7 +75,7 @@ export class CustomQuery {
         : "";
     this.medicationRequestQuery =
       rxnormFilter !== ""
-        ? `/MedicationRequest?subject=${patientId}&code=${rxnormFilter}&_include=MedicationRequest:medication&_include=MedicationRequest:medication.administration`
+        ? `/MedicationRequest?subject=${patientId}&code=${rxnormFilter}&_include=MedicationRequest:medication&_include=MedicationRequest:intended-performer`
```

Comment:
For some reason, Public HAPI didn't like `medication.administration` as an includes param, which was breaking the inclusion of the medication requests for the patient. I did a bit of sparse googling and _think_ this is the closest of the provided options below from the error message but if someone knows better lmk and I'm happy to change it.

```
Invalid _include parameter value: &quot;MedicationRequest:medication.administration&quot;. Unknown search parameter &quot;medication.administration&quot; for resource type &quot;MedicationRequest&quot;. Valid search parameters for this search are: [encounter, intended-dispenser, intended-performer, medication, patient, requester, subject]
```

Reply:
Ah I didn't see this before during my original review. I can take a look at it tomorrow morning and see what I come up with

Reply:
From what I could gather, the `administration` property seems more like how the dosage is administered. See the table here https://build.fhir.org/medicationrequest.html#bnr . Some of the examples given are taking an oral tablet, receiving a shot, etc. On the other hand, `intended-performer` is the individual who actually applies the medication to the patient (evidenced by the `Reference` property pointing to the patient, a physician, etc.). Unfortunately none of these actually relate to how the medication is taken--dispenser refers to the filling pharmacy, performer to the one giving the drug, etc. `encounter` _might_ be relevant as another event-type resource, but I don't love the fit of any of them :/

Reply:
Thanks for the clarification Brandon! Will note that in the current cut of main medication requests seem to be broken because of this issue, so even without an exact match this will be an improvement over what exists. Would appreciate follow-ups to make this work as intended though!

Reply:
I looked into this a little more and I think we can update the query to look for this instead: `&_revinclude=MedicationAdministration:request`, which should pull back the Medication Administrations associated with the request.